src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.de.xlf

```diff
@@ -1564,12 +1564,12 @@
       </trans-unit>
       <trans-unit id="PreferIsEmptyOverCountDescription">
         <source>For determining whether the object contains or not any items, prefer using 'IsEmpty' property rather than retrieving the number of items from the 'Count' property and comparing it to 0 or 1.</source>
-        <target state="translated">Wenn Sie bestimmen möchten, ob das Objekt Elemente enthält oder nicht, verwenden Sie die Eigenschaft "IsEmpty", anstatt die Anzahl von Elementen aus der Eigenschaft "Count" abzurufen und mit 0 oder 1 zu vergleichen.</target>
+        <target state="translated">Um zu bestimmen ob das Objekt Elemente enthält verwenden Sie die Eigenschaft "IsEmpty", anstatt die Anzahl von Elementen aus der Eigenschaft "Count" abzurufen und mit 0 oder 1 zu vergleichen.</target>
         <note />
       </trans-unit>
       <trans-unit id="PreferIsEmptyOverCountMessage">
-        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object contains or not any items</source>
-        <target state="translated">Verwenden Sie anstelle von "Count" die Eigenschaft "IsEmpty", um festzulegen, ob das Objekt Elemente enthält oder nicht.</target>
+        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object is empty</source>
+        <target state="translated">Verwenden Sie die Eigenschaft "IsEmpty" anstelle von "Count", um zu bestimmen, ob das Objekt Elemente enthält</target>
```

Comment:
I don't think we should do translations manually

Reply:
@buyaa-n ok, since you don't want to have manual translations I've reduced this PR to changing the english text in the resx and then re-generating the xlf files (which re-sets the state to "needs-review-translation").

Reply:
Thanks @BrunoJuchli, JFYI it is not because i don't like your translation, even if your translation was correct for this language the other languages will be left without translation/update. We normally just add/update the English version and build will update the state so that localization team translate/update them